### DISCUSSION TITLE
Remove noreferrer attribute from <LinkOverlay isExternal />

### DISCUSSION THREAD

**Discussion by albertovaccarella** [chakra-ui/chakra-ui#8642] [Ideas]
### Description

The <LinkOverlay> component adds a noreferrer attribute when you include the isExternal attribute. I think we should remove this since it prevents the Referer header from being sent on link clicks to external sites, the same issue was already reported and fixed for the <Link> component (see issue #4243 ).

### Problem Statement/Justification

`noreferrer` does not add anything in terms of security since the introduction of `noopener`, which is, correctly, already included when `isExternal` is set.

### Proposed Solution or API

Remove `noreferrer` from this line:
https://github.com/chakra-ui/chakra-ui/blob/58fb61d92b3247593e2682e9001c607f51861f33/packages/react/src/components/link/link-box.tsx#L24C1-L24C53

### Additional Information

- #4243 
- https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer
- https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

**Comment by segunadebayo** [comment]
`LinkOverlay` and `isExternal` are v2. Use `Link` with `asChild`.
